build.sbt

```diff
@@ -121,8 +122,25 @@ def formattingPreferences = {
     .setPreference(SpacesAroundMultiImports, true)
 }
 
+val defaultMultiJvmOptions: List[String] = {
+  import scala.collection.JavaConverters._
+  // multinode.D= and multinode.X= makes it possible to pass arbitrary
+  // -D or -X arguments to the forked jvm, e.g. -Dmultinode.Xmx512m
```

Comment:
Mmmm, I'm failing to see the relation between the example, and the initial sentence "multinode.D= and multinode.X= makes it possible ..."